src/Components/Todo_list/Components/Modal.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import './Todo_list.css'
+
+class Modal extends React.PureComponent{
+
+    state={
+        status: '',
+        value: ''
+    }
+    render(){
+        const { handleToggleModal, handleSaveTask } = this.props
+
+        return <div className='Modal_Content'>
+         <h1>New Task</h1>
+         <div>
+            <div className="Checkbox_Id">
+            <label htmlFor="todo" onClick={ () => {this.setState({status:'todo' })}}><input name='new'  type='radio' id='todo' className='label'/>
```

Comment:
```suggestion
            <label htmlFor="todo" onClick={ () => {this.setState({status:'todo' })}}>
              <input name='new' type='radio' id='todo' className='label' />
```

Không được `setState` trong hàm `render`

```
onClick={ () => {this.setState({status:'todo' })}}
```
Sửa thành
```
onClick={ () => this.setSomthing({ status: 'todo' }) }
```